src/Runtime/OMTensorList.inc

```diff
@@ -26,25 +26,21 @@
 #endif
 
 #include "onnx-mlir/Runtime/OMTensorList.h"
+#include <string.h>
```

Comment:
What's this for?

Reply:
Probably from before…. I can remove this include.

Reply:
And it's back again. It's for `memcpy`.